release/Magarena/scripts/Azcanta__the_Sunken_Ruin.groovy

```diff
@@ -2,8 +2,9 @@ def action = {
     final MagicGame game, final MagicEvent event ->
     final MagicCardList topCards = new MagicCardList(event.getRefCardList());
     event.processChosenCards(game, {
-        topCards.removeCard(it);
+        game.doAction(new RevealAction(it));
```

Comment:
LookAction is sufficient.

Reply:
Sorry, my bad, this card needs a Reveal as it stated in the oracle text. This is correct.